### PR TITLE
Add pull request template.

### PR DESCRIPTION
**Pull request by josemauro**
This PR add a pull request template to be showed by GitHub when a pull request is being created. This improvement solves the issue #2  .

### CODE REVIEW

**Review by gleybersonandrade — APPROVED** [review]

File: PULL_REQUEST_TEMPLATE.md
Comment (hdiogenes) [inline PULL_REQUEST_TEMPLATE.md]: > If the pull request does not address an existing issue with the appropriated label (e.g. bug label)

...then what?

Comment (josemauro) [reply PULL_REQUEST_TEMPLATE.md]: The complete phrase is "If the pull request does not address an existing issue with the "bug" label, the maintainers have the final say on whether the current behavior is a bug." I'll remove the incomplete phrase because the Note in this paragraph summarizes the missing idea.

File: PULL_REQUEST_TEMPLATE.md
Comment (hdiogenes) [inline PULL_REQUEST_TEMPLATE.md]: Please rephrase, there are two "notes" together.

Comment (josemauro) [reply PULL_REQUEST_TEMPLATE.md]: Done!

**Pull request merged**
